pages/api/job/[id].js

```diff
@@ -0,0 +1,57 @@
+import dbConnect from '../../../utils/dbConnect'
```

Comment:
![image](https://user-images.githubusercontent.com/62170666/129684635-19977d0b-8fb7-431f-b592-78477312690e.png)
double check the path

Reply:
had this error before can fix it